src/ctim/schemas/vocabularies.cljc

```diff
@@ -259,7 +259,7 @@
     "cisco_mid"
     "cisco_uc_id"          ;; Cisco Unified Connector ID
     "cisco_cm_id"          ;; Cisco NVM UUID
-    "device"               
+    "device"
```

Comment:
we need the process observables here

Reply:
They are still there, I did not remove them.

Reply:
@jyoverma I think you got confused, when you check a PR on github, the interface is showing you what changed only and not the full source files every time, hence the observables are still here, if you want to make sure select "view file" in the interface.

Reply:
Ok in that case, I approve these changes